cmake/configs/nuttx_crazyflie_default.cmake

```diff
@@ -100,6 +100,7 @@ set(config_module_list
 	lib/version
 	lib/DriverFramework/framework
 	platforms/nuttx
+	lib/micro-CDR
```

Comment:
We need to be careful which base dependencies we introduce and why. I checked and the reason here is that the generated source for the uorb topic serialization depends on it. As the code is not used, it will be removed when linking, so the binary should be the same. It will however increase compilation time.
@davids5 fyi